bot/config.py.sample

```diff
@@ -104,4 +104,14 @@ z['scheduler'] = {
     'friday': '20:00',
 }
 
+# ****************************************************************************
+# Post run configuration
+#
+# To do any post run tasks. It runs 'script' and followed by 'cleanup'
+# ****************************************************************************
+z['postrun'] = {
+    'script': '/path/to/script',
```

Comment:
I see you are running 'script' right after 'cleanup' so I guess you could simply provide a list of paths to scripts and just run all of those in for loop.
z['postrun'] = ['/path/to/script', '/path/to/cleanup/script']

This way you will not duplicate the code [1]

Reply:
Good idea.